src/kraken/std/protobuf/__init__.py

```diff
@@ -0,0 +1,15 @@
+from typing import Any
+
+from kraken.core.api import Project
+
+from .tasks.buf_task import BufFormatTask, BufLintTask
+
+
+def bufFormat(*, name: str = "buf.format", project: Project | None = None, **kwargs: Any) -> BufFormatTask:
```

Comment:
PEP8 recommends snake case

```suggestion
def buf_format(*, name: str = "buf.format", project: Project | None = None, **kwargs: Any) -> BufFormatTask:
```